e2etests/run_athens.go

```diff
@@ -0,0 +1,56 @@
+// +build e2etests
+
+package e2etests
+
+import (
+	"fmt"
+	"net/http"
+	"os/exec"
+	"path"
+	"path/filepath"
+	"time"
+)
+
+func buildAthens(goBin string, destPath string, env []string) (string, error) {
+	target := path.Join(destPath, "athens-proxy")
+	binFolder, err := filepath.Abs("../cmd/proxy")
+	if err != nil {
+		return "", fmt.Errorf("Failed to get athens source path %v", err)
+	}
+
+	cmd := exec.Command(goBin, "build", "-o", target, binFolder)
+	cmd.Env = env
+	err = cmd.Run()
+	return target, err
+}
+
+func stopAthens() error {
+	cmd := exec.Command("pkill", "athens-proxy")
+	err := cmd.Run()
+	return err
+}
+
+func runAthensAndWait(athensBin string, env []string) error {
+	cmd := exec.Command(athensBin)
+	cmd.Env = env
+
+	go func() {
+		cmd.Run()
+	}()
+
+	ticker := time.NewTicker(time.Second)
+	timeout := time.After(2 * time.Minute)
```

Comment:
two minutes seems a lil crazy to wait for athens to start 😄 -- let's make it 20 seconds?

Reply:
Sorry, I am getting deviated to k8s timescales